synthesizer/tests/test_process_execute.rs

```diff
@@ -39,7 +39,9 @@ fn test_process_execute() {
         // Run the test.
         let output = run_test(process.clone(), test);
         // Check against the expected output.
-        test.check(&output).unwrap();
+        if let Err(err) = test.check(&output) {
```

Comment:
This changes the behavior of the integration test framework a bit.
Should we be saving the output if check errors?